### DISCUSSION TITLE
A few things I noticed while I was using Saber

### DISCUSSION THREAD

**Discussion by ceskyDJ** [saber-notes/saber#834] [Ideas]
I have a Saber v0.14.9 from Google Play on my ~Samsung Galaxy S22~ Samsung Galaxy Tab S8 with Android 13. I tried Saber for taking a mind map (classical programmer notes for better understanding the problem and finding its solution...). I found a few problems noted in next paragraphs. @adil192, could you look at them, please? I don't create issues, because the problems may need to be discussed more deeply (I don't know, how you see these things, etc.).

1. I don't see currently active color anywhere, so I have to try it and erase it. It's a pretty annoying after some time.
2. SPen eraser don't work for me (it does nothing), but it should word, shouldn't it?
3. There is no visual divider between a canvas and a left side menu, so I don't know, where I can write and where I can't.
4. When I draw from the left side using SPen (finger drawing is disabled), it opens side menu, so my stroke can't be completed.

**Comment by ceskyDJ** [comment]
Ad 4:
https://github.com/adil192/saber/assets/48548230/60ce43b9-78ad-46e9-81da-8e9e3aa5b8b8

**Comment by adil192** [comment]
1. I assume you mean without opening the color bar, right? I could maybe replace the current paint pallet icon with the selected color. 
2. It only works if you're already touching the screen before you press the eraser button. See https://github.com/adil192/saber/pull/783#issuecomment-1627527498
3. 
4. I believe this is simply because you're triggering the Android gesture to exit fullscreen. There should be an option in system Settings to prevent the S Pen triggering navigation events.

Besides that, please create issues for 1 and 3.

**Reply by ceskyDJ** [reply]
> 1. I assume you mean without opening the color bar, right? I could maybe replace the current paint pallet icon with the selected color.

Yes, exactly. I don't know, how to do that. Samsung Notes has a little detail on pen/highlighter. So, maybe just change color of a spike of the pen/highlighter? It could look good and very minimalist (non-disruptively). What do you think?

> 2. It only works if you're already touching the screen before you press the eraser button. See https://github.com/adil192/saber/pull/783#issuecomment-1627527498

Hmm, It's working now. So, ignore this thing, please.

> 4. I believe this is simply because you're triggering the Android gesture to exit fullscreen. There should be an option in system Settings to prevent the S Pen triggering navigation events.

I know, I have turned off these navigation gestures for SPen.

> Besides that, please create issues for 1 and 3.

Ok.

**Reply by ceskyDJ** [reply]
> Besides that, please create issues for 1 and 3.

Done :D.

**Reply by adil192** [reply]
> > 4. I believe this is simply because you're triggering the Android gesture to exit fullscreen. There should be an option in system Settings to prevent the S Pen triggering navigation events.
> 
> I know, I have turned off these navigation gestures for SPen.

Just to clarify, does 4 still occur when you've turned off the navigation gestures for S Pen?

**Reply by ceskyDJ** [reply]
I've turned off this feature a long time ago. So, yes, it's still on the table. At least I disabled gestures like "Go home", "Go back" and "Open task manager". Is there any other way how to disable more gestures on Android?
